src/wallet/rpc/wallet.cpp

```diff
@@ -582,6 +582,94 @@ static RPCHelpMan upgradewallet()
     };
 }
 
+RPCHelpMan simulaterawtransaction()
+{
+    return RPCHelpMan{"simulaterawtransaction",
+        "\nCalculate the balance change resulting in the signing and broadcasting of the given transaction(s).\n" +
+        HELP_REQUIRING_PASSPHRASE,
```

Comment:
In ae8564c198a3f9c57cc896489afcb78d6b947b45 "rpc/wallet: add simulaterawtransaction RPC"

The wallet does not need to be unlocked, so this help text is not needed.

Reply:
Good point, removed.